src/app/[locale]/components/AddGroupToUser.tsx

```diff
@@ -0,0 +1,93 @@
+"use client";
+
+import { useForm } from "react-hook-form";
+import { useDispatch, useSelector } from "react-redux";
+import { selectAuth } from "../../../redux/slices/authSlice";
+import { useRouter } from "next/navigation";
+import { useTranslations } from "next-intl";
+import { useEffect, useState } from "react";
+import { AppDispatch } from "../../../redux/store";
+import { getAllGroups, selectGroup } from "../../../redux/slices/groupSlice";
+import { addUserGroup, getUser } from "../../../redux/slices/userSlice";
+import { GroupData } from "../types/typescript";
+
+export default function AddGroupToUser({
+  locale,
+  userId,
+}: {
+  locale: string;
+  userId: string;
+}) {
+  const router = useRouter();
+  const t = useTranslations("translations");
+  const [warning, setWarning] = useState<string>();
+  const dispatch = useDispatch<AppDispatch>();
+
+  const { isLoggedIn } = useSelector(selectAuth);
+  const { groups } = useSelector(selectGroup);
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const handleAddGroup = async (data: GroupData) => {
+    if (!data._id) {
+      setWarning("select group");
```

Comment:
isn't setWarning redundant here since react-hook-form can handle validation errors and display appropriate feedback

Reply:
issue fixed ,  setWarning removed